development-vm/README.md

```diff
@@ -1,276 +1,212 @@
+### Set your Git username and email
 
-You're likely on the production VPN. Disconnect the VPN and `reload`
-your VM.
+You can assign your name and email to commits on the VM:
 
-#### Vagrant error :NFS is reporting that your exports file is invalid
-```
-==> default: Exporting NFS shared folders...
-NFS is reporting that your exports file is invalid. Vagrant does
-this check before making any changes to the file. Please correct
-the issues below and execute "vagrant reload":
+    dev$ git config --global user.email "friendly.giraffe@digital.cabinet-office.gov.uk"
+    dev$ git config --global user.name "Friendly Giraffe"
 
-exports:2: path contains non-directory or non-existent components: /Users/<username>/path/to/vagrant
-exports:2: no usable directories in export entry
-exports:2: using fallback (marked offline): /
-exports:5: path contains non-directory or non-existent components: /Users/<username>/path/to/vagrant
-exports:5: no usable directories in export entry
-exports:5: using fallback (marked offline): /
-```
+## 5. Set up your apps
 
-This means that you may already have old vagrant path definitions in your `/etc/exports` file.
+Begin by checking out all of the gov.uk services. There's a handy shortcut:
 
-Try opening up `/etc/exports` file to identify old or unwanted vagrant paths and removing them if necessary
+    dev$ cd /var/govuk/govuk-puppet/development-vm
+    dev$ ./checkout-repos.sh < alphagov_repos
 
-On opening `/etc/exports` file each set begins with # VAGRANT-BEGIN: and ends with # VAGRANT-END:. Make sure to delete these and any other lines between VAGRANT-BEGIN: and VAGRANT-END:
+Most of our apps are written in Ruby and use [Bundler](http://bundler.io/rationale.html) to manage their dependencies. To boot apps, you’ll also need to install those dependencies:
 
-or maybe
+    dev$ ./update-bundler.sh
 
-```
-sudo rm /etc/exports
-sudo touch /etc/exports
+There are also some Python apps, which use [PIP](https://pip.pypa.io/en/stable/). You’ll probably need to install those dependencies too, so run:
 
-vagrant halt
-vagrant up
-```
+    dev$ ./update-pip.sh
 
-### Errors installing vagrant-dns
+> `~/govuk/` on your host machine is mounted as `/var/govuk` inside the VM. Any app repositories you clone should go here.
 
-Installing vagrant-dns with `vagrant plugin install vagrant-dns` against Vagrant 1.9 installed may give an error like:
+## 6. Access remote environments
 
-```
-/opt/vagrant/embedded/lib/ruby/2.2.0/rubygems/dependency.rb:315:in `to_specs': Could not find 'celluloid' (>= 0.16.0) among 45 total gem(s) (Gem::LoadError)
-```
+### Access servers via SSH
 
-It looks like this might be a problem with Vagrant 1.9.0, because installing 1.8.6 fixes the problem. The issue has been raised with vagrant-dns, so they may have a better workaround: https://github.com/BerlinVagrant/vagrant-dns/issues/45
+By now hopefully your pull request from earlier will have been merged. It's time to test your access to servers via SSH.
```

Comment:
Your pull request from earlier will hopefully have been merged by now. It's time to test your access to servers via SSH.